sources/Outputs/include/Dyd.h

```diff
@@ -121,12 +124,24 @@ class Dyd {
   /**
    * @brief Create black box model for generator
    *
+   * @param collection the collection for which the connections will be added
    * @param def generator definition to use
    * @param basename basename for file
    *
    * @returns black box model for generator
```

Comment:
incorrect comment

Reply:
How is it incorrect ?

Reply:
Your function doesn't return a black box model (remark overriden by the remark about about the logic of the function)